### ISSUE TITLE
[Bug report] Allow to specify custom path to store converted csv->hdf5 files

### ISSUE THREAD

**Issue by lrq3000** [vaexio/vaex#1054]
When using `vaex.from_csv('somefile.csv', convert=True)`, the converted hdf5 file will always be stored in the same folder as the csv file. This may be inadequate if the user wants to keep separate the raw original data from the postprocessed hdf5 files. There is also the use case where the user wants to store the converted hdf5 files on another disk than the raw csv data (maybe to have more storage space?).

The argument `convert` accepts a path as a `str` instead of a boolean value (see [here](https://github.com/vaexio/vaex/blob/3a0785cf9265cec615c470172c8cd460c3bd17a8/packages/vaex-core/vaex/__init__.py#L538)) but it seems it's only meant to replace the `filename_or_buffer` argument (so I'm not sure the purpose?). It would be maybe more useful if the path would actually determine the folder where the converted files are meant to be stored to?

**Comment by JovanVeljanoski** [comment]
Hi @lrq3000 

The behaviour your describe is exactly the behaviour that was intended. It looks like a bug indeed. I also found a related bug in `vaex.open(...)`. I will open a PR addressing this shortly. 
Thank you!

(let's keep this open until this is solved).

**Comment by lrq3000** [comment]
That is a crazingly fast reply and PR! Thank you for having fixed it already!

**Comment by byaminov** [comment]
Oh indeed, thanks for reporting! This workaround should help until the fix is released:
```
with open('somefile.csv') as f:
    vaex.from_csv(f, convert='/mnt/other/folder/converted.hdf5')
```

**Comment by maartenbreddels** [comment]
This should be fixed in #1055, if not please reopen.

Released, to install:
```
$ pip install "vaex-core>=4.0.0a11"
# or for the while kitchsink:
$ pip install "vaex>=4.0.0a6"
```
